addons/medical_treatment/functions/fnc_placeInBodyBagOrGrave.sqf

```diff
@@ -0,0 +1,58 @@
+#include "script_component.hpp"
+/*
+ * Author: Glowbal, drofseh
+ * Places a dead body inside a body bag.
+ *
+ * Arguments:
+ * 0: Arguments <ARRAY>
+ *   0: Medic <OBJECT>
+ *   1: Patient <OBJECT>
+ * 1: Resting Place Object <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "ACE_bodyBagObject"] call ace_medical_treatment_fnc_placeInBodyBagOrGrave
+ *
+ * Public: No
+ */
+
+params ["_args", "_restingPlaceObject"];
+_args params ["_medic", "_patient"];
+TRACE_1("placeInBodyBagOrGrave",_patient);
+
+if (!local _patient) exitWith {
+    TRACE_1("Calling where local",local _patient);
+    [QGVAR(placeInGrave), [nil, _patient], _patient] call CBA_fnc_targetEvent;
```

Comment:
```suggestion
    [QGVAR(placeInBodyBagOrGrave), _this, _patient] call CBA_fnc_targetEvent;
```

Why was `nil` passed instead of `_medic`?

Reply:
No idea, predates my changes.